### PR TITLE
[#162565304] Add additional info section for ownership listing

### PR DESCRIPTION
**Pull request by software-project**
https://www.pivotaltracker.com/n/projects/1405352/stories/162565304

WIP. Waiting for Realtor Commission section to be finalize.
@akegan @chrisdolendo @callaghanc is this something we would want to test with new approach?

### CODE REVIEW

**Comment by akegan** [comment]
cc @jrubenoff re: realtor commission content

**Comment by jrubenoff** [comment]
Let's use the content in these mockups for now. (@akegan, I'm presenting these to SMEs on Thursday.)

### If the commission unit is "flat fee"

screen shot 2019-02-05 at 6 36 05 pm

### If the commission unit is "percentage"

screen shot 2019-02-05 at 6 27 04 pm

### If there's no commission

screen shot 2019-02-05 at 6 35 27 pm

**Comment by jrubenoff** [comment]
Update: that realtor commission content is final! MOHCD approved it on Thursday.

File: app/services/force/listing_service.rb [hunk 1]
```diff
@@ -180,11 +184,8 @@ def self.array_sort!(listing)
       # Add stubbed listing fields
       # rubocop:disable LineLength
       stubbed_listing_data = {
-        'Allows_Realtor_Commission' => true,
-        'Realtor_Commission_Percentage' => 15,
-        'Realtor_Commission_Info' => 'TBD but this will probably be a shortish string',
         'CC_and_R_URL' => 'http://www.google.com',
-        'Repricing_Mechanism' => 'TODO: Replace this with a real example of a repricing mechanism. Here\'s some sample text with links <a href=\"http://sf-moh.org/index.aspx?page=295\" target=\"_blank\">Inclusionary Affordable Housing Program Monitoring and Procedures Manual 2013</a>',
+        'Repricing_Mechanism' => 'TODO: Replace this with a real example of a repricing mechanism. Here\'s some sample text with links <a href="http://sf-moh.org/index.aspx?page=295" target="_blank">Inclusionary Affordable Housing Program Monitoring and Procedures Manual 2013</a>',
```
Comment (akegan) [inline app/services/force/listing_service.rb]: uh oh, does this mean that this field is missing?

Comment (akegan) [reply app/services/force/listing_service.rb]: nevermind, I see that you were just changing the formatting. I went ahead and removed the fields from this PR that we're getting from salesforce now.

**Comment by akegan** [comment]
I updated the cc&r url to open in a new page, but now I think @software-project or @callaghanc  should review my changes before we merge the PR if possible

**Comment by software-project** [comment]
@akegan it looks good, so I'm happy to merge it when story is accepted on pivotal

**Comment by akegan** [comment]
@software-project these fields are just used for listing details page so they don't need to be whitelisted for browse

**Comment by software-project** [comment]
@akegan all good then, thanks

**Comment by akegan** [comment]
just going to do one last QA then I'll merge

**Pull request merged**